File: Task/FizzBuzz/F-Sharp/fizzbuzz-2.fs
[1..100]
|> List.map (fun x ->
            match x with
            | _ when x % 15 = 0 ->"fizzbuzz"
            | _ when x % 5 = 0 -> "buzz"
            | _ when x % 3 = 0 -> "fizz"
            | _ ->  x.ToString())
|> List.iter (fun x -> printfn "%s" x)

File: Task/FizzBuzz/Zen-C/fizzbuzz-2.c
fn main() {
    for i in 1..=100 {
        match i % 15 {
            0           => { println "FizzBuzz" },
            3, 6, 9, 12 => { println "Fizz" },
            5, 10       => { println "Buzz" },
            _           => { println "{i}" },
        }
    }
}
